src/Utils/errors.tsx

```diff
@@ -4,22 +4,26 @@ export class NetworkError extends Error {
   response: any
 }
 
-interface ErrorInfo {
-  componentStack: string
+export class ErrorWithMetadata extends Error {
+  metadata: object
+  decoratedMessage: string
+  constructor(message, metadata = {}) {
+    super(message)
+    this.decoratedMessage = message + " | " + JSON.stringify(metadata)
```

Comment:
Is `this.decoratedMessage` being used anywhere?

Reply:
Ah, I see, its being passed along within `this` down below.

Reply:
No, not currently -- it'd be reasonable just to get rid of it until we need it.